gemelli/ctf.py

```diff
@@ -3,8 +3,8 @@
 from pandas import concat
 from pandas import DataFrame
 from skbio import OrdinationResults, DistanceMatrix
-from gemelli.factorization import TensorFactorization
-from gemelli.preprocessing import build, rclr
+from gemelli.tensor_factorization import TensorFactorization
```

Comment:
`factorization` -> `tensor_factorization` and `rclr` -> `tensor_rclr` are kind of huge breaking API changes. Definitely requires a major version.

I would be worried if now `gemelli.factorization` now automatically refers to matrix factorization or something that. If anyone uses your API, the error message they see will be along the lines of "`TensorFactorization` does not exist in `gemelli.factorization`" as opposed to "`TensorFactorization` has moved from `gemelli.factorization` to `gemelli.tensor_factorization`"

Reply:
Reverted the changes back to `factorization `